rest-tests/src/test/java/io/syndesis/qe/RestTestHooks.java

```diff
@@ -83,4 +90,19 @@ public void resetPostgresDB() {
         log.info("Back default values of PostgresDB");
         TestSupport.getInstance().resetDB();
     }
+
+    @After("@operator-addons-jaeger-external")
+    public void undeployJaeger() throws TimeoutException, InterruptedException {
+        log.info("Undeploying external Jaeger");
+        ResourceFactory.get(Jaeger.class).undeploy();
+        ResourceFactory.get(Syndesis.class).updateAddon(Addon.JAEGER, false);
+        OpenShiftWaitUtils.waitForPodIsReloaded("syndesis-server");
+        log.info("Setting default syndesis-jaeger addon");
+        Map<String, Object> properties = new HashMap<>();
+        properties.put("clientOnly", false);
+        properties.put("operatorOnly", false);
+        ResourceFactory.get(Syndesis.class).updateAddon(Addon.JAEGER, true, properties);
+        ResourceFactory.get(Syndesis.class).jaegerWorkarounds();
+        OpenShiftWaitUtils.waitForPodIsReloaded("syndesis-server");
+    }
```

Comment:
since every test cleans the namespace, this shouldn't be necessary or?

Reply:
Clean namespace is not able to delete external-jaeger pods and undeploySyndesis() failed on timeout.
https://fuse-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/fuse-7.6-syndesis-tests-operator/88/Cucumber_20report_20-_20operator/

However, it is true that `ResourceFactory.get(Jaeger.class).undeploy();` can be enough in case the next test delete the namespace. (otherwise the Syndesis will be in inconsistent state)